internal/cli/events/list.go

```diff
@@ -43,6 +43,10 @@ func (opts *ListOpts) initStore() error {
 	return err
 }
 
+var listTemplate = `ID	TYPE_NAME	CREATED{{range .Results}}
```

Comment:
I left you a similar comment on another PR but we can simplified this to make it more human
```suggestion
var listTemplate = `ID	TYPE	CREATED{{range .Results}}
```